services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -126,7 +128,7 @@ protected void doControllerMsg(CommandMessage message) {
 
     private void handleCommand(CommandMessage message, String replyToTopic, Destination replyDestination)
             throws FlowCommandException {
-
+        logger.info("Request Received {}, {}, {}", message, replyToTopic, replyDestination);
```

Comment:
You have been asked to remove this log message, at least 2 times.

Reply:
Updated